sdk/playwrighttesting/Azure.Developer.MicrosoftPlaywrightTesting.TestLogger/src/PlaywrightService.cs

```diff
@@ -187,25 +191,37 @@ private void InitializePlaywrightServiceEnvironmentVariables(string? os = null,
         }
         if (!string.IsNullOrEmpty(os))
         {
-            Environment.SetEnvironmentVariable(ServiceEnvironmentVariable.PlaywrightServiceOs, os);
+            Os = os;
+            if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable(ServiceEnvironmentVariable.PlaywrightServiceOs)))
```

Comment:
in case of os is not null and env variable is set then value of Os and env would be different, Do we need this to happen ? if not, can we always make sure value of Os is updated to correct value ?